ppr-ui/src/store/getters/state-getters.ts

```diff
@@ -212,6 +212,20 @@ export const isMhrRegistration = (state: StateIF): boolean => {
     APIRegistrationTypes.MANUFACTURED_HOME_REGISTRATION
 }
 
+/** Is true when all steps are valid as well as staff payment and authorization are valid */
+export const isMhrRegistrationReviewValid = (state: StateIF): boolean => {
+  return state.stateModel.mhrValidationState.reviewConfirmValid.authorizationValid &&
+  state.stateModel.mhrValidationState.reviewConfirmValid?.staffPaymentValid &&
+  useMhrValidations(toRefs(getMhrRegistrationValidationModel(state)))
```

Comment:
`toRefs(getMhrRegistrationValidationModel(state))` could be a const on its own, so we don't have call it each time.

Reply:
Changes made :)

Reply:
Much cleaner